examples/LDA.jl

```diff
@@ -1,4 +1,4 @@
-using TopicModels, Plots, UMAP
+using TopicModels, Plots
```

Comment:
Yes, but since there is no "suggests" functionality in the Pkg ecosystem, it seemed better style to omit packages that aren't required at compile time.  I've noticed other projects (eg https://turing.ml/dev/tutorials/3-bayesnn/) take the same approach regarding deps that exist solely in examples from the documentation.  Does that seem reasonable?